app/src/main/java/com/babylon/wallet/android/presentation/main/MainScreen.kt

```diff
@@ -37,16 +37,19 @@ fun MainScreen(
                 onNavigateToMnemonicRestore = onNavigateToMnemonicRestore
             )
         }
+
         is AppState.IncompatibleProfile -> {
-            LaunchedEffect(state.initialAppState) {
```

Comment:
@jakub-rdx Hmm definitely Im not an expert here and not saying it is incorrect, but Is there an actual difference now?
with Unit it will be called only once (when entered composition) and before only when state.initialAppState changed but since its wrapped with **when** it is actually the same. Is my understanding correct ?